README.md

```diff
@@ -81,7 +74,8 @@ source .envrc
 However, this is only recommended on CI workflows since it makes the environment changes persistent in your current shell session.
 
 ## Build Dependencies
-This is work in progress.
+
+This is a work in progress.
```

Comment:
I think the previous text here was correct, so I'd remove the "a" again but the additional newline can stay